ihatemoney.wsgi

```diff
@@ -0,0 +1,9 @@
+import sys
+import os
+
+__HERE__ = os.path.dirname(os.path.abspath(__file__))
+
+# Add the budget directory to the path so we can then import from run
+sys.path.insert(0, os.path.join(__HERE__, 'budget'))
```

Comment:
Once #193 is ok, and when we will be pypi released, that `sys.path` extension will become useless. Maybe you could add some reminder comment about it in the source code.